.azure-pipelines/unit_test.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+trigger:
+  - main
+
+pool:
+  vmImage: ubuntu-latest
+strategy:
+  matrix:
+    Python37:
+      python.version: '3.7'
+    Python38:
+      python.version: '3.8'
+
+steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(python.version)'
+    inputs:
+      versionSpec: '$(python.version)'
+
+  - script: |
+      python3 -m pip install --upgrade flake8 prometheus-client opencensus opencensus-ext-azure
```

Comment:
Can you add psutil as well? It was added to /src/worker/exporters/install/common.sh recently? Is used by node_exporter.py. We will be working on a more standardized way for installing dependencies

Reply:
sure, addressed.